tbx/project_styleguide/templates/patterns/molecules/pagination/pagination.html

```diff
@@ -2,35 +2,41 @@
     <nav aria-label="Pagination" class="pagination">
         <ul class="pagination__list">
             {% if pages.has_previous %}
+                {# Previous page link #}
                 <li class="pagination__page">
                     <a href="?page={{ pages.previous_page_number }}{% if extra_url_params %}&amp;{{ extra_url_params }}{% endif %}" class="pagination__link pagination__link--previous" rel="prev" aria-label="Go to previous page">
                         {% include "patterns/atoms/icons/icon.html" with name="arrow" classname="pagination__prev-icon" %}
-                        Previous
+                        <span class="hidden md:inline">Previous</span>
                     </a>
                 </li>
-                {% if pages.number > 3 %}
-                    <li class="pagination__page">
+                {# First page link - always shows if this is not the first page, but the logic here stops it displaying twice if it is one of the numbers on either side of the current page  #}
+                {% if pages.number > 2 %}
+                    <li class="pagination__page hidden lg:block">
                         <a href="?page=1{% if extra_url_params %}&amp;{{ extra_url_params }}{% endif %}" class="pagination__link" aria-label="Page 1 of {{ pages.paginator.num_pages }}">
                             1
                         </a>
                     </li>
-                    {% if pages.number > 4 %}
-                        <li class="pagination__page">
+                    {# Ellipsis between the first page and the page before the current page  #}
+                    {% if pages.number > 3 %}
+                        <li class="pagination__page hidden lg:block" aria-label="dot dot dot">
```

Comment:
Interestingly Gov UK don't add an aria-lablel to this and it's announced 'midline horizontal ellipsis'. I'm not sure what would be most useful to the user here.

Reply:
https://design-system.service.gov.uk/components/pagination/for-list-pages/

Reply:
Their code is `<li class="govuk-pagination__item govuk-pagination__item--ellipses">&ctdot;</li>`

Reply:
Ah interesting - I originally copied this from RNIB, which didn't have it either, but when I tested in Safari it read it out as something like 'doooooot'. Maybe I'll remove thought to be in line with best practice.

Reply:
I've updated to use `&#8230;` for the text, but I've left the aria-label in - reason being, without it, the screen reader announces *only* where you are in terms of the list items, e.g. '3 of 7' which is confusing in the context of 'page 4 of 9' or whatever. It still reads that out afterwards, but at least this way it announces 'elipsis' first.

Reply:
Amende https://github.com/torchbox/torchbox.com/pull/53/commits/a782d823f0cf4c91d2a0c9cc92c26d36eafd0997